polls.php

```diff
@@ -255,16 +255,22 @@
 			$options[$i] = '';
 		}
 
-		if(trim($options[$i]) != "")
+		if($mybb->settings['polloptionlimit'] != 0 && my_strlen($options[$i]) > $mybb->settings['polloptionlimit'])
 		{
-			$optioncount++;
+			$lengtherror = 1;
+			break;
 		}
 
-		if(my_strlen($options[$i]) > $mybb->settings['polloptionlimit'] && $mybb->settings['polloptionlimit'] != 0)
+		if(strpos($options[$i], '||~|~||') !== false)
 		{
-			$lengtherror = 1;
+			$sequenceerror = 1;
```

Comment:
Well, I can change it, but there are lots of variables with words not separated with `_` even in the nearby of this line. I don't think there should be problems with reading it.